testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -556,4 +558,42 @@ public void testMaterializedView()
 
         assertUpdate("DROP MATERIALIZED VIEW " + viewName);
     }
+
+    @Test
+    public void testCommentView()
+    {
+        if (hasBehavior(SUPPORTS_CREATE_VIEW) && !hasBehavior(SUPPORTS_COMMENT_ON_VIEW)) {
+            String viewName = "nation_v";
```

Comment:
Please note that we avoid using static identifiers because it will lead to flaky tests when the test runs on SaaS connectors.